extensions/verifier/src/verifier/transcript.rs

```diff
@@ -52,7 +64,10 @@ impl<S> Transcript<G1Affine, OpenVmLoader> for OpenVmTranscript<G1Affine, S, Vec
             .collect_vec();
         let hash = keccak256(&data);
         self.buf = hash.to_vec();
-        OpenVmScalar(Fr::from_be_bytes(&hash), PhantomData)
+        let fr: Halo2Fr = u256_to_fe(U256::from_be_bytes(hash));
+        use halo2curves_axiom::ff::PrimeField;
+        let bytes: [u8; 32] = fr.to_repr();
+        OpenVmScalar(Fr::from_le_bytes(&bytes), PhantomData)
```

Comment:
I think you can do `Fr::from_be_bytes(&hash)` and then `.reduce()` to ensure its canonical

Reply:
oh good.. i did not notice that function